src/components/studies/app-design/AppDesign.tsx

```diff
@@ -610,9 +610,9 @@ const AppDesign: React.FunctionComponent<
       const formattedPhone = formatPhoneNumber(phoneWithoutZipcode)
       setIrbPhoneNumber(formattedPhone)
     }
-    const isWelcomeScreenDataIsEmpty =
+    const isWelcomeScreenDataEmpty =
       Object.keys(study.clientData.welcomeScreenData || {}).length == 0
-    const welcomeScreenData = isWelcomeScreenDataIsEmpty
+    const welcomeScreenData = isWelcomeScreenDataEmpty
```

Comment:
can it just be `welcomeScreenData  =  Object.keys(study.clientData.welcomeScreenData || {}).length == 0`